### PR TITLE
Created Bartender object, added styling

### PR DESCRIPTION
**Pull request by strothj**

### CODE REVIEW

**Comment by annejones817** [comment]
I think this looks really good! My only stylistic suggestion is that you may want to hide the questions once the user clicks "make my drink". It took me a minute to realize I needed to scroll to actually see my drink once I clicked the button (but of course, this would vary based on screen size). Good job!

File: index.html [hunk 1]
```diff
@@ -10,18 +10,25 @@
...
-    <main>
+    <main class='main'>
         <article>
             <section>
                 <form id="js-form">
                     <!--Populated by app.js-->
                 </form>
             </section>
+            <section>
+                <h2 class="hidden" id="js-drink-header">Your drink!</h2>
```
Comment (annejones817) [inline index.html:27]: Styling - inconsistent quote type -- above, you use single quotes

**Review by annejones817 — COMMENTED** [review]
Just had one styling comment on inconsistent quote types in your HTML. Good job!

File: index.html (hunk 1 above)
Comment (strothj) [reply index.html:27]: Wow, good catch. I didn't realize I was doing that. Habit from coding in JavaScript with single quotes.

**Pull request merged**